src/commands/deviceprofiles/translations.ts

```diff
@@ -0,0 +1,101 @@
+import { type ArgumentsCamelCase, type Argv, type CommandModule } from 'yargs'
+
+import { type DeviceProfileTranslations, type LocaleReference } from '@smartthings/core-sdk'
+
+import { fatalError } from '../../lib/util.js'
+import {
+	apiOrganizationCommand,
+	apiOrganizationCommandBuilder,
+	type APIOrganizationCommandFlags,
+} from '../../lib/command/api-organization-command.js'
+import {
+	outputItemOrList,
+	outputItemOrListBuilder,
+	type OutputItemOrListConfig,
+	type OutputItemOrListFlags,
+} from '../../lib/command/listing-io.js'
+import { chooseDeviceProfileFn } from '../../lib/command/util/deviceprofiles-choose.js'
+import { buildTableOutput } from '../../lib/command/util/deviceprofiles-translations-table.js'
+
+
+export type CommandArgs =
+	& APIOrganizationCommandFlags
+	& OutputItemOrListFlags
+	& {
+		profileIdOrIndex?: string
+		tag?: string
+		verbose: boolean
+	}
+
+const command = 'deviceprofiles:translations [profileIdOrIndex] [tag]'
+
+const describe = 'list locales translated for a device profile or display translations for a specific locale'
+
+const builder = (yargs: Argv): Argv<CommandArgs> =>
+	outputItemOrListBuilder(apiOrganizationCommandBuilder(yargs))
+		.positional(
+			'profile-id-or-index',
+			{ describe: 'device profile id or number from list', type: 'string' },
+		)
+		.positional('tag', { describe: 'the locale tag or number from list', type: 'string' })
+		.option('verbose',
+			{ alias: 'v', describe: 'include list of locales in table output', type: 'boolean', default: false })
+		.example([
+			[
+				'$0 deviceprofiles:translations',
+				'prompt for a device profile and list its locales',
+			],
+			[
+				'$0 deviceprofiles:translations --verbose',
+				'including locales with profiles when prompting for a device profile',
```

Comment:
To be consistent with other example text, I think it would be better for this to say "include locales"

Reply:
Um, I think you mean, "to not sound ridiculous, ...". 😂 Fixed.